### PR TITLE
Update desktop-app-changelog.rst

### PR DESCRIPTION
**Pull request by amyblais**
Add known issues

### CODE REVIEW

File: source/install/desktop-app-changelog.rst [hunk 1]
```diff
@@ -52,6 +52,7 @@ All Platforms
 - Shortened the maximum length (width) for server tab names to 224px.
 - Updated the menu bar and system tray icons for improved contrast.
 - Removed ``libappnotify1`` as a dependency requirement in Debian installers as it's no longer shipped in Debian's Bullseye. It's still recommended to install where available.
+- "Minimize to tray" setting was removed.
```
Comment (Willyfrog) [inline source/install/desktop-app-changelog.rst:55]: is it minimize to tray or start minimized?

File: source/install/desktop-app-changelog.rst [hunk 2]
```diff
@@ -92,9 +93,13 @@ All Platforms
 Known Issues
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 - The new spellchecker connects to Google servers for downloading updated dictionaries.
+- Unread messages icon is missing from the taskbar on Windows following 4.7.0 upgrade `MM-37807 <https://mattermost.atlassian.net/browse/MM-37807>`_.
+- Taskbar icon flashes on new message arrival on Windows despite setting being disabled `MM-37797 <https://mattermost.atlassian.net/browse/MM-37797>`_.
+- "Add Server" appears on each startup on servers with GPO `MM-37863 <https://mattermost.atlassian.net/browse/MM-37863>`_.
 - Clicking on **View > Find** doesn't work `MM-36606 <https://mattermost.atlassian.net/browse/MM-36606>`_.
 - Right click menu is missing from the ``jira connect`` modal `MM-36032 <https://mattermost.atlassian.net/browse/MM-36032>`_.
 - Search field is focused on first start of the app `MM-35249 <https://mattermost.atlassian.net/browse/MM-35249>`_.
+- The app crashes when downloading a zip file. **Ticket?**
```
Comment (esethna) [inline source/install/desktop-app-changelog.rst:102]: @Willyfrog do you know if we have a ticket for this?

Comment (Willyfrog) [reply source/install/desktop-app-changelog.rst:102]: I don't see any reports of this, but community is not working properly ATM, any link so I can check?

Comment (amyblais) [reply source/install/desktop-app-changelog.rst:102]: @Willyfrog I added you to the channel just now where it was reported (the latest messages there).

File: source/install/desktop-app-changelog.rst (hunk 1 above)
Comment (amyblais) [reply source/install/desktop-app-changelog.rst:55]: The setting described here is "minimizeToTray" https://github.com/mattermost/desktop/issues/1687, but not sure if it's the official name.

File: source/install/desktop-app-changelog.rst (hunk 2 above)
Comment (amyblais) [reply source/install/desktop-app-changelog.rst]: I removed it for now until it is officially reproduced. I couldn't repro on Mac / Win.

File: source/install/desktop-app-changelog.rst (hunk 1 above)
Comment (esethna) [reply source/install/desktop-app-changelog.rst:55]: @Willyfrog @amyblais I'm not clear on what actually changed here.

**Review by esethna — APPROVED** [review]

File: source/install/desktop-app-changelog.rst (hunk 1 above)
Comment (amyblais) [reply source/install/desktop-app-changelog.rst]: I'll remove it for now.

**Pull request merged**